src/DotNetBridge/NativeDataInterop.cs

```diff
@@ -516,19 +516,19 @@ public override void Set()
                         {
                             for (int i = 0; i < _buffer.Length; i++)
                             {
-                                _poker(_buffer.Values[i], _colIndex + i, _input.Position);
+                                _poker(_buffer.GetValues()[i], _colIndex + i, _input.Position);
                             }
                         }
                         else
                         {
                             int ii = 0;
                             for (int i = 0; i < _buffer.Length; i++)
                             {
-                                while (ii < _buffer.Count && _buffer.Indices[ii] < i)
+                                while (ii < _buffer.GetValues().Length && _buffer.GetIndices()[ii] < i)
```

Comment:
>GetValues [](start = 52, length = 9)

It may be cleaner to add two lines before the for loop:
var values=_buffer.GetValues();
var indices=_buffer.GetIndices();

Reply:
Changed

---
In reply to: [248379904](https://github.com/Microsoft/NimbusML/pull/77#discussion_r248379904) [](ancestors = 248379904)